src/site/filters/liquid.js

```diff
@@ -19,7 +19,34 @@ module.exports = function registerFilters() {
   liquid.filters.humanizeTimestamp = dt =>
     moment.unix(dt).format('MMMM D, YYYY');
 
-  liquid.filters.formatDate = (dt, format) => moment(dt).format(format);
+  liquid.filters.timeZone = (dt, tz, format) => {
+    if (dt && tz) {
+      const tzOffset = new Date(dt).getTimezoneOffset(tz) * 60000;
+      const dtDate = new Date(
+        dt.toLocaleString('en-US', {
+          timeZone: tz,
+        }),
+      ).getTime();
+
+      const diffToMoment = dtDate - tzOffset;
+
+      const prettyTime = moment(diffToMoment).format(format);
+      const prettyTimeFormatted = prettyTime
+        .replace(/AM/g, 'a.m.')
+        .replace(/PM/g, 'p.m.');
+
+      return prettyTimeFormatted;
+    }
+    return dt;
+  };
+
+  liquid.filters.formatDate = (dt, format) => {
```

Comment:
Will this refactor effect other elements that are using this filter?

Reply:
It will, and this is intentional. Ryan said that a.m. / p.m. is to be applied everywhere. USDS style guide calls for this treatment.